src/manual/Installing/Installing-from-source-Unix.md

```diff
@@ -7,7 +7,7 @@ The following are instructions to build ClamAV *version 0.104 and newer* using C
 - [Building ClamAV with CMake (v0.104 and newer)](#building-clamav-with-cmake-v0104-and-newer)
   - [Install prerequisites](#install-prerequisites)
     - [Alpine](#alpine)
-    - [Redhat / Centos / Fedora](#redhat--centos--fedora)
```

Comment:
We don't need to remove references to Redhat just because we don't do automated testing on Redhat.